mbuild/lib/recipes/polymer.py

```diff
@@ -364,6 +389,54 @@ def add_end_groups(
             else:
                 raise ValueError("Label must be 'head' or 'tail'")
 
+    def create_periodic_bond(self, port1, port2, axis="z"):
```

Comment:
remove `port1` and `port2` as parameters, we can use `self.head_port` and `self.tail_port`